ipatests/test_integration/test_sssd.py

```diff
@@ -403,3 +404,135 @@ def test_trustdomain_disable_disables_subdomain(self):
+                                         'child1', '--groups', 'parent'])
+        assert 'Group name: child1' in cmd_output.stdout_text
+        assert 'Member groups: parent' in cmd_output.stdout_text
+        assert 'Number of members added 1' in cmd_output.stdout_text
+
+        cmd_output = master.run_command(['ipa', 'group-add-member',
+                                         'child2', '--groups', 'child1'])
+        assert 'Group name: child2' in cmd_output.stdout_text
+        assert 'Member groups: child1' in cmd_output.stdout_text
+        assert 'Indirect Member groups: parent' in cmd_output.stdout_text
+
+        # add user to group 'parent'
+        cmd_output = master.run_command(['ipa', 'group-add-member',
+                                         'parent', '--users', username])
+        assert 'Group name: parent' in cmd_output.stdout_text
+        assert 'Member users: {}'.format(username) in cmd_output.stdout_text
+        assert 'Member of groups: child1' in cmd_output.stdout_text
+        assert 'Indirect Member of group: child2' in cmd_output.stdout_text
+
+        # clear sssd_cache
+        clear_sssd_cache(master)
+
+        # user lookup
+        cmd_output = master.run_command(['ipa', 'user-show', username])
+        assert 'parent' in cmd_output.stdout_text
+        assert 'ipausers' in cmd_output.stdout_text
+        assert 'child1' in cmd_output.stdout_text
+        assert 'child2' in cmd_output.stdout_text
+
+        clear_sssd_cache(master)
+
+        # ssh to client using the user created above from master
+        cmd_output = master.run_command(['ssh', '-q',
+                                        '%s@%s' % (username, client.hostname),
+                                         'groups'], stdin_text=userpasswd)
+        assert '{}'.format(username) in cmd_output.stdout_text
+        assert 'a' in cmd_output.stdout_text
+        assert 'b' in cmd_output.stdout_text
+        assert 'c' in cmd_output.stdout_text
+
+        # add member
+        cmd_output = master.run_command(['ipa', 'group-add-member',
+                                         'child1', '--users', username])
+        assert 'Group name: child1' in cmd_output.stdout_text
+        assert 'Member users: {}'.format(username) in cmd_output.stdout_text
+        assert 'Member groups: parent' in cmd_output.stdout_text
+        assert 'Member of groups: child2' in cmd_output.stdout_text
+        assert 'Number of members added 1' in cmd_output.stdout_text
+
+        # now check ssh on the client
+        clear_sssd_cache(client)
+        cmd_output = client.run_command(['ssh', '-q',
+                                        '%s@%s' % (username, client.hostname),
+                                         'groups'], stdin_text=userpasswd)
+        assert '{}'.format(username) in cmd_output.stdout_text
+        assert 'a' in cmd_output.stdout_text
+        assert 'b' in cmd_output.stdout_text
+        assert 'c' in cmd_output.stdout_text
```

Comment:
Need to replace a, b and c with parent, child1 and child2

Reply:
replaced it